core/domain/contribution_stats_services.py

```diff
@@ -236,7 +238,8 @@ def get_translation_submitter_total_stats(
     language_code: str,
     sort_by: Optional[str],
     topic_ids: Optional[List[str]],
-    max_days_since_last_activity: Optional[int]
+    start_date: datetime.date,
+    end_date: datetime.date
```

Comment:
You missed this part from my comment
> You want to get some data for some range - then pass the range, a single object of structure with two fields: start_date, end_date.

Create a single object that represents the date range. Also look in the codebase, isn't there one already?

Reply:
Sorry @nikitaevg, I might be missing something because I didn't find any such object. If possible could you please tell me the file where this object is defined. However I found a similar date range functionality in a certificate component, there are two variables `to_date` and `from_date` which are similar to `start_date` and `end_date`. PTAL

Reply:
If you don't have such an object - just create one. Sorry, I didn't want to give you the impression that there is definitely such an object. Create a class Range that will have two fields and it might also have some utility functions, like check intersection of ranges, or check that a particular date is in that range.